README.md

```diff
@@ -55,6 +55,16 @@ Please refer to the following instructions to setup Oppiabot for the first time
   ```
 
 4. The Oppiabot uses environment variables. These are configured in the server settings. To deploy the bot locally, create a `.env` file and copy the contents of `.env.example` to it. You will need to adjust these variables accordingly following the instructions in the subsequent steps. Run following command to copy `.env.example` to `.env`
+
+If using windows cmd, go to oppiabot folder type
+``` bash 
+echo "%cd%"
+```
+The above command will give you <b>path</b>
```

Comment:
```suggestion
The above command will give you <b><path></b>, then type:
```

Reply:
Removed it as now using a direct path instead of absolute as suggested in the previous review.Thanks.